common/src/main/java/com/wynntils/utils/wynn/LocationUtils.java

```diff
@@ -37,15 +37,15 @@ public static Matcher strictCoordinateMatcher(String str) {
     }
 
     public static void shareLocation(String target) {
-        String locationString = "My location is at [" + (int) McUtils.player().position().x + ", "
```

Comment:
Why these changes removing `[]`? Coordinates in Wynncraft are presented in brackets, so I think we should do the same.

Reply:
The coord feature already adds then, so this was just a "double" bracket

Reply:
Aha!

Reply:
... uh, but wait. 

This is added only if the recipient is also running Wynntils.

The proper fix should be that the coord highligher feature should only add `[...]` if there is not one already.

Reply:
I mean, I am not sure brackets are essential if the coordinate is not clickable

Reply:
It's back to what I said before: "Coordinates in Wynncraft are presented in brackets, so I think we should do the same." Consistency for the user experience. This was the reason I implemented the message this way long time ago.

Reply:
Hmm, I do agree that it's nice to keep consistent with Wynn where we can, but I'd argue it's not necessary here as this is just a convenient way for people to send their location, it's not a server message.

When non-Wynntils users see players sending coords in chat normally they'll just see "go to x,y,z" for example, no brackets so I don't think we need a special case for this. Unless we start adding brackets everytime a user sends coords

Reply:
Ok, let me turn this around the other way. I still think we should not add `[...]` around coordinates if there already are brackets. Let's say Wynncraft sends:

"Go back to [123,65,-789]"

Then we should not add additional brackets.

And if we do add this functionality, we can just as well keep the brackets here, to clearly mark the coordinates. I think of it as being a "good citizen", and try to always present coordinates in the same style.

Reply:
Ok, I've just added them back and it'll remove the brackets if it finds coords with them at the start and end